tests/foreman/cli/test_computeresource_rhev.py

```diff
@@ -16,31 +16,80 @@
 :Upstream: No
 """
 import pytest
+from attrdict import AttrDict
 from fauxfactory import gen_string
+from nailgun import entities
+from wrapanapi import RHEVMSystem
 
+from robottelo.api.utils import configure_provisioning
 from robottelo.cli.computeresource import ComputeResource
 from robottelo.cli.factory import CLIFactoryError
 from robottelo.cli.factory import CLIReturnCodeError
 from robottelo.cli.factory import make_compute_resource
+from robottelo.cli.factory import make_host
 from robottelo.cli.factory import make_os
+from robottelo.cli.host import Host
+from robottelo.cli.settings import Settings
 from robottelo.config import settings
+from robottelo.constants import RHEL_6_MAJOR_VERSION
+from robottelo.constants import RHEL_7_MAJOR_VERSION
+from robottelo.helpers import host_provisioning_check
+from robottelo.utils.issue_handlers import is_open
 
 pytestmark = pytest.mark.on_premises_provisioning
 
 
 @pytest.fixture(scope='module')
 def rhev():
-    rhev = type('rhev', (object,), {})()
-    rhev.current_rhev_url = settings.rhev.hostname
-    rhev.username = settings.rhev.username
-    rhev.password = settings.rhev.password
-    rhev.datacenter = settings.rhev.datacenter
-    rhev.image_arch = settings.rhev.image_arch
-    rhev.image_uuid = settings.rhev.image_uuid
-    rhev.os = make_os()
+    bridge = settings.vlan_networking.bridge
+    rhev = AttrDict()
```

Comment:
why create an attrdict instance when it will be overwritten by a box object a line later?